Fusetools 3478 reactivate test and provide specific camel xsd version for test resources

apupier opened the pull request:
requires https://github.com/jbosstools/jbosstools-fuse/pull/1695

lhein commented:
The naming of this PR does not reflect all the changes done here. You did reactivate tests but you also changed a lot of schema locatiions.

apupier commented:
> The naming of this PR does not reflect all the changes done here. You did reactivate tests but you also changed a lot of schema locatiions or is this PR based on another?

The PR was based on another.
There are also xsd location s to update that are used in UI tests. i updated the title.
